src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Meals Service</title>
+
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    </head>
+    <body class="px-4">
+        <header>
+            <a href="javascript:void(0);" class="text-decoration-none">
+                <img id="iconImg" alt="My Logo">
+            </a>
+            <a href="#meals" class=" px-3 text-decoration-none text-dark">Meals</a>
+            <a href="#meals" class=" px-3 text-decoration-none text-dark">Categories</a>
+            <a href="#meals" class=" px-3 text-decoration-none text-dark">Search by Name</a>
+        </header>
```

Comment:
- [ ] According to the requirements:

![image](https://github.com/alexansaa/Meals/assets/77069011/ef1b016b-74da-44a7-b208-c5399ca5ff58)

Please make sure to include functions for your counters. Currently, you haven't implemented any of them! 🤓 🔧 


Home page counter:
![image](https://github.com/alexansaa/Meals/assets/77069011/3a919cb5-9819-4568-a270-c8c5f73b6555)

Comments counter:
![image](https://github.com/alexansaa/Meals/assets/77069011/24bbab33-aa11-4497-af72-475ff66b99c6)

has a static `7`:
![image](https://github.com/alexansaa/Meals/assets/77069011/2dc3609a-1d9f-4f15-81e8-934dcf5c6117)

Please consider working on this, and don't forget to test them, this is another **important** requirement! 🤓 🔧